opencti-platform/opencti-graphql/src/domain/user.js

```diff
@@ -198,19 +198,28 @@ export const batchGroups = async (context, user, userId, opts = {}) => {
   return batchListThroughGetTo(context, user, userId, RELATION_MEMBER_OF, ENTITY_TYPE_GROUP, opts);
 };
 
-const internalUserIds = Object.keys(INTERNAL_USERS);
+// build only a creator object with what we need to expose of users
+const buildCreatorUser = (user) => {
+  if (!user) {
+    return user;
+  }
```

Comment:
yes, I understand the question, there are different possibilities here, I could have returned null, or false, but I thought that since it's a method that transforms the user to extract only the fields we need, if the user is not defined, I just return the user without change.